app/lib/max_allowed_days_loader.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module MaxAllowedDaysLoader
+  module_function
+
+  def load_account_suspension
```

Comment:
Could it be just `account_suspension` instead of `load_account_suspension`? (Same for `account_inactivity` and `contract_unpaid_time`.)

Reply:
i was just about to write that.. don't use `prefix` as `send_`  `load_` etc.